src/plugins/dashboard/public/application/top_nav/dashboard_top_nav.tsx

```diff
@@ -591,9 +607,9 @@ export function DashboardTopNav({
             primaryActionButton: (
               <PrimaryActionButton
                 label={i18n.translate('dashboard.solutionToolbar.addPanelButtonLabel', {
-                  defaultMessage: 'Create panel',
+                  defaultMessage: 'Create chart',
                 })}
-                onClick={createNew}
+                onClick={createNewVisType(lensAlias)}
```

Comment:
Can this button take a `href`?

Reply:
What is the benefit to adding an `href` here? Are you suggesting using an href instead of an `onClick` handler?

I don't think just having an href is sufficient bc we need to use `navigateToEditor` from the embeddable state transfer service to handle the by-value embeddable `Save and return` workflow.

Reply:
I think you are seeing some of the same limitations as we did in Lens, and we haven't been able to find a better way: https://github.com/elastic/kibana/pull/96723#issuecomment-818856940